src/LogicalCameraSensor.cc

```diff
@@ -56,6 +60,8 @@ class gz::sensors::LogicalCameraSensorPrivate
 
   /// \brief Msg containg info on models detected by logical camera
   msgs::LogicalCameraImage msg;
+
+  msgs::LogicalCameraSensor msg_logic;
```

Comment:
```suggestion
  /// \brief Msg containing logical camera frustum info.
  public: msgs::LogicalCameraSensor msgLogic;
```

Looks like we forgot to add `public` to `msg` above as well.

Reply:
Updated!